<!DOCTYPE html>


<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    
    <title>Lecture 1 &mdash; Genome Sciences 541 2014.04.17 documentation</title>
    
    <link rel="stylesheet" href="_static/basic.css" type="text/css" />
    <link rel="stylesheet" href="_static/styles.css" type="text/css" />
    <link rel="stylesheet" href="_static/slides.css" type="text/css" />
    

    <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
    
    <script type="text/javascript">
      var DOCUMENTATION_OPTIONS = {
        URL_ROOT:    './',
        VERSION:     '2014.04.17',
        COLLAPSE_INDEX: false,
        FILE_SUFFIX: '.html',
        HAS_SOURCE:  true
      };
    </script>
    <script type="text/javascript" src="_static/jquery.js"></script>
    <script type="text/javascript" src="_static/underscore.js"></script>
    <script type="text/javascript" src="_static/doctools.js"></script>
    <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
    <script type="text/javascript" src="_static/common.js"></script>
    <script type="text/javascript" src="_static/slides.js"></script>
    <script type="text/javascript" src="_static/sync.js"></script>
    <script type="text/javascript" src="_static/controller.js"></script>
    <script type="text/javascript" src="_static/init.js"></script>
    
    <link rel="top" title="Genome Sciences 541 2014.04.17 documentation" href="index.html" />
    <link rel="prev" title="Genome Sciences 541" href="index.html" /> 
  </head>
  <body>

<section
   id="slide_container"
   class='slides layout-regular'>


  <article class="slide level-1" id="lecture-1">
<h1>Lecture 1</h1>
<ul class="simple">
<li>Genome Sciences 541, <a class="reference external" href="http://research.fhcrc.org/bloom/en.html">Jesse Bloom</a></li>
<li>These slides are at <a class="reference external" href="http://jbloom.github.io/GenomeSciences541/lecture_1.html">http://jbloom.github.io/GenomeSciences541/lecture_1.html</a></li>
<li>Reading for this lecture is <a class="reference download internal" href="_downloads/Zuckerkandl_and_Pauling-1965.pdf"><tt class="xref download docutils literal"><span class="pre">Zuckerkandl</span> <span class="pre">and</span> <span class="pre">Pauling</span> <span class="pre">(1965)</span></tt></a>.
Read pages 97-114 and 147-152 carefully,
skim pages 115-146,
feel free to skip pages 153-end.</li>
<li>The homework for this lecture is at ???.</li>
</ul>

</article>
<article class="slide level-2" id="lecture-1-the-evolution-of-protein-coding-genes">
<h2>Lecture 1: the evolution of protein-coding genes</h2>
<ul class="simple">
<li><a class="reference download internal" href="_downloads/Zuckerkandl_and_Pauling-1965.pdf"><tt class="xref download docutils literal"><span class="pre">Zuckerkandl</span> <span class="pre">and</span> <span class="pre">Pauling</span> <span class="pre">(1965)</span></tt></a>: one of the first molecular studies of protein evolution</li>
<li>Structure imposes many constraints</li>
<li>Very hard to quantitatively understand these constraints</li>
<li>Sequence divergence <span class="math">\(\ne\)</span> Functional divergence</li>
<li>A molecular clock?</li>
</ul>

</article>
<article class="slide level-3" id="genes-evolve-at-the-level-of-dna-or-sometimes-rna">
<h3>Genes evolve at the level of DNA (or sometimes RNA)</h3>
<p>We routinely abstract the complex molecule DNA to strings of letters with little loss of relevant information:</p>
<div class="highlight-python"><div class="highlight"><pre>atg gtg ctc agc gag gga gaa tgg cag ttg gtt ctg cac gtc ...
</pre></div>
</div>
<p><em>It has not escaped our notice that the specific pairing we have postulated immediately suggests a possible copying mechanism for the genetic material.</em> -- <a class="reference external" href="http://www.nature.com/nature/dna50/watsoncrick.pdf">Watson and Crick (1953)</a></p>
<p><em>The translation from nucleic acid to protein proceeds in a sequential fashion according to a systematic code with relatively simple rules.</em> -- <a class="reference external" href="http://www.nobelprize.org/nobel_prizes/medicine/laureates/1968/nirenberg-lecture.pdf">Nirenberg (1968)</a></p>

</article>
<article class="slide level-3" id="but-most-selection-is-on-the-protein">
<h3>But most selection is on the protein</h3>
<p>The properties of proteins are <strong>not</strong> easily abstracted.</p>
<p>Proteins are linear polymers encoded by DNA:</p>
<div class="highlight-python"><div class="highlight"><pre>atg gtg ctc agc gag gga gaa tgg cag ttg gtt ctg cac gtc ...
M   V   L   S   E   G   E   W   Q   L   V   L   H   V   ...
</pre></div>
</div>
<p>But proteins derive their relevant properties from their three-dimensional structures.</p>

</article>
<article class="slide level-3" id="properties-of-proteins-are-not-easily-abstracted">
<h3>Properties of proteins are not easily abstracted</h3>
<div class="figure align-center">
<a class="reference internal image-reference" href="_images/myoglobin_static.png"><img alt="_images/myoglobin_static.png" src="_images/myoglobin_static.png" style="width: 61%;" /></a>
<p class="caption"><a class="reference external" href="https://github.com/jbloom/GenomeSciences541/tree/gh-pages/source/myoglobin_structure">PyMol analysis of sperm whale myoglobin</a></p>
<div class="legend">
<em>Perhaps the most remarkable features of the molecule are its complexity and its lack of symmetry. The arrangement seems to be almost totally lacking in the kind of regularities which one instinctively anticipates, and it is more complicated than has been predicated by any theory of protein structure.</em> -- <a class="reference external" href="http://www.chem.uwec.edu/Chem452_Media/pages/readings/media/Kendrew_et_al_1958.pdf">Kendrew et al (1958)</a></div>
</div>

</article>
<article class="slide level-3" id="context-for-reading-zuckerkandl-and-pauling-1965">
<h3>Context for reading Zuckerkandl and Pauling (1965)</h3>
<div class="figure align-center">
<a class="reference internal image-reference" href="_images/Zuckerkandl_and_Pauling-1965.png"><img alt="_images/Zuckerkandl_and_Pauling-1965.png" src="_images/Zuckerkandl_and_Pauling-1965.png" style="width: 90%;" /></a>
<p class="caption"><a class="reference download internal" href="_downloads/Zuckerkandl_and_Pauling-1965.pdf"><tt class="xref download docutils literal"><span class="pre">Zuckerkandl</span> <span class="pre">and</span> <span class="pre">Pauling</span> <span class="pre">(1965)</span></tt></a></p>
</div>

</article>
<article class="slide level-3" id="they-knew-the-structure-of-sperm-whale-myoglobin">
<h3>They knew the structure of sperm whale myoglobin</h3>
<div class="figure align-center">
<a class="reference internal image-reference" href="_images/myoglobin_static.png"><img alt="_images/myoglobin_static.png" src="_images/myoglobin_static.png" style="width: 90%;" /></a>
<p class="caption"><a class="reference external" href="https://github.com/jbloom/GenomeSciences541/tree/gh-pages/source/myoglobin_structure">PyMol analysis of sperm whale myoglobin</a></p>
<div class="legend">
Structure had been determined by <a class="reference external" href="http://www.chem.uwec.edu/Chem452_Media/pages/readings/media/Kendrew_et_al_1958.pdf">Kendrew et al (1958)</a></div>
</div>

</article>
<article class="slide level-3" id="they-knew-that-single-mutations-could-potentially-have-large-biological-consequences">
<h3>They knew that single mutations could potentially have large biological consequences</h3>
<div class="highlight-python"><div class="highlight"><pre><span class="n">MVHLTPEEKSAVT</span><span class="o">...</span>
<span class="n">MVHLTPvEKSAVT</span><span class="o">...</span>
</pre></div>
</div>
<p><a class="reference external" href="http://www.sciencemag.org/content/110/2865/543">&quot;Sickle Cell Anemia, a Molecular Disease&quot; (Pauling et al, 1949)</a></p>
<p><a class="reference external" href="http://www.nature.com/nature/journal/v178/n4537/abs/178792a0.html">&quot;A Specific Chemical Difference between Globins of Normal and Sickle-cell Anemia Hemoglobins&quot; (Ingram, 1956)</a></p>

</article>
<article class="slide level-3" id="they-knew-the-sequences-of-a-variety-of-homologous-globins">
<h3>They knew the sequences of a variety of homologous globins</h3>
<ul class="simple">
<li>sperm whale myoglobin</li>
<li>human myoglobin</li>
<li>human hemoglobin <span class="math">\(\alpha\)</span>, <span class="math">\(\beta\)</span>, <span class="math">\(\gamma\)</span>, and <span class="math">\(\delta\)</span> chains</li>
<li>horse hemoglobin <span class="math">\(\alpha\)</span> and <span class="math">\(\beta\)</span> chains</li>
<li>cattle hemoglobin <span class="math">\(\alpha\)</span> and <span class="math">\(\beta\)</span> chains</li>
<li>cattle fetal hemoglobin chain</li>
<li>pig hemoglobin <span class="math">\(\alpha\)</span> and <span class="math">\(\beta\)</span> chains</li>
<li>gorilla <span class="math">\(\alpha\)</span> and <span class="math">\(\beta\)</span> chains</li>
<li>carp hemoglobin <span class="math">\(\alpha\)</span> chain</li>
<li>lamprey hemoglobin chain</li>
</ul>

</article>
<article class="slide level-3" id="analysis-of-myoglobin-homologs">
<h3>Analysis of myoglobin homologs</h3>
<p>We will perform an analysis similar to that of Zuckerkandl and Pauling, and look at some of the conclusions that they draw.</p>

</article>
<article class="slide level-3" id="myoglobin-homologs">
<h3>Myoglobin homologs</h3>
<div class="highlight-python"><div class="highlight"><pre>&gt;carp
MA----DHELVLKCWGGVEADFEGTGGEVLTRLFKQHPETQKLFPKFVGIA-QSDLAGNAAVKAHGATVLKSWASCLKARGDHAAILKPLATTHANTHKIALNNFRLITEVLVKVMAEKAGLD--AGGQSALRRVMDVVIGDIDTYYKEIGFAG
&gt;chicken
MGLSDQEWQQVLTIWGKVEADIAGHGHEVLMRLFHDHPETLDRFDKFKGLKTPDQMKGSEDLKKHGATVLTQLGKILKQKGNHESELKPLAQTHATKHKIPVKYLEFISEVIIKVIAEKHAADFGADSQAAMKKALELFRNDMASKYKEFGFQG
&gt;cow
MGLSDGEWQLVLNAWGKVEADVAGHGQEVLIRLFTGHPETLEKFDKFKHLKTEAEMKASEDLKKHGNTVLTALGGILKKKGHHEAEVKHLAESHANKHKIPVKYLEFISDAIIHVLHAKHPSDFGADAQAAMSKALELFRNDMAAQYKVLGFHG
&gt;horse
MGLSDGEWQQVLNVWGKVEADIAGHGQEVLIRLFTGHPETLEKFDKFKHLKTEAEMKASEDLKKHGTVVLTALGGILKKKGHHEAELKPLAQSHATKHKIPIKYLEFISDAIIHVLHSKHPGDFGADAQGAMTKALELFRNDIAAKYKELGFQG
&gt;human
MGLSDGEWQLVLNVWGKVEADIPGHGQEVLIRLFKGHPETLEKFDKFKHLKSEDEMKASEDLKKHGATVLTALGGILKKKGHHEAEIKPLAQSHATKHKIPVKYLEFISECIIQVLQSKHPGDFGADAQGAMNKALELFRKDMASNYKELGFQG
&gt;mouse
MGLSDGEWQLVLNVWGKVEADLAGHGQEVLIGLFKTHPETLDKFDKFKNLKSEEDMKGSEDLKKHGCTVLTALGTILKKKGQHAAEIQPLAQSHATKHKIPVKYLEFISEIIIEVLKKRHSGDFGADAQGAMSKALELFRNDIAAKYKELGFQG
&gt;pig
MGLSDGEWQLVLNVWGKVEADVAGHGQEVLIRLFKGHPETLEKFDKFKHLKSEDEMKASEDLKKHGNTVLTALGGILKKKGHHEAELTPLAQSHATKHKIPVKYLEFISEAIIQVLQSKHPGDFGADAQGAMSKALELFRNDMAAKYKELGFQG
&gt;sperm-whale
MVLSEGEWQLVLHVWAKVEADVAGHGQDILIRLFKSHPETLEKFDRFKHLKTEAEMKASEDLKKHGVTVLTALGAILKKKGHHEAELKPLAQSHATKHKIPIKYLEFISEAIIHVLHSRHPGDFGADAQGAMNKALELFRKDIAAKYKELGYQG
&gt;zebrafish
MA----DHDLVLKCWGAVEADYAANGGEVLNRLFKEYPDTLKLFPKFSGIS-QGDLAGSPAVAAHGATVLKKLGELLKAKGDHAALLKPLANTHANIHKVALNNFRLITEVLVKVMAEKAGLD--AAGQGALRRVMDAVIGDIDGYYKEIGFAG
</pre></div>
</div>
<p>Sequences at <a class="reference external" href="https://github.com/jbloom/GenomeSciences541/tree/gh-pages/source/myoglobin_homologs">Analysis of myoglobin homologs</a></p>

</article>
<article class="slide level-3" id="site-variability-among-myoglobin-homologs">
<h3>Site variability among myoglobin homologs</h3>
<div class="figure align-center">
<a class="reference internal image-reference" href="_images/myoglobin_site_preferences_logoplot.jpg"><img alt="_images/myoglobin_site_preferences_logoplot.jpg" src="_images/myoglobin_site_preferences_logoplot.jpg" style="width: 91%;" /></a>
<p class="caption">Code /data to produce this figure at <a class="reference external" href="https://github.com/jbloom/GenomeSciences541/tree/gh-pages/source/myoglobin_homologs">Analysis of myoglobin homologs</a></p>
</div>

</article>
<article class="slide level-3" id="very-little-is-strictly-conserved">
<h3>Very little is strictly conserved</h3>
<div class="figure align-center">
<a class="reference internal image-reference" href="_images/invariant_sites.png"><img alt="_images/invariant_sites.png" src="_images/invariant_sites.png" style="width: 90%;" /></a>
<p class="caption"><a class="reference download internal" href="_downloads/Zuckerkandl_and_Pauling-1965.pdf"><tt class="xref download docutils literal"><span class="pre">Zuckerkandl</span> <span class="pre">and</span> <span class="pre">Pauling</span> <span class="pre">(1965)</span></tt></a></p>
</div>

</article>
<article class="slide level-3" id="the-evolution-studied-in-molecular-phylogenetics-is-due-to-both-mutation-and-selection">
<h3>The evolution studied in molecular phylogenetics is due to both mutation and selection</h3>
<div class="highlight-python"><div class="highlight"><pre>atg gtg ctc agc gag gga gaa tgg cag ttg gtt ctg cTc gtc ...
atg gtg ctc agc gag gga gaa tgA cag ttg gtt ctg cac gtc ...
atg gtg ctc agc Cag gga gaa tgg cag ttg gtt ctg cac gtc ...
</pre></div>
</div>
<div class="highlight-python"><div class="highlight"><pre>atg gtg Atc agc gag gga gaa tgg cag ttg gtt ctg cac gtc ...
atg gtg ctc agc Cag gga gaa tgg cTg ttg gtt ctg cac gtc ...
atg gtg ctc agc Cag gga gaa tgg cag ttg gtA ctg cac gtc ...
</pre></div>
</div>
<div class="highlight-python"><div class="highlight"><pre>atg gtA ctc agc Cag gga gaa tgg cag ttg gtt ctg cac gtc ...
atg gtg ctc agc Cag gga gaa tgg cTg ttg gtt ctg Aac gtc ...
atg gtg ctc agc Cag gga gaa tgg cTg ttg gtA ctg cac gtc ...
</pre></div>
</div>
<div class="highlight-python"><div class="highlight"><pre>atg atg cAc agc Cag gga gaa tgg cTg ttg gtt ctg cac gtc ...
atg gtg ctc agc Cag gga gaa tgg cTg ttg gtt ctg Aac gtc ...
atg gtg ctc agc Cag gga gaa tgg cTg ttg gtA ctg cac gtc ...
</pre></div>
</div>

</article>
<article class="slide level-3" id="more">
<h3>More</h3>
<p>In these lectures (and in most molecular phylogenetics), we consider only point substitutions::
<a class="reference download internal" href="_downloads/Zuckerkandl_and_Pauling-1965.pdf"><tt class="xref download docutils literal"><span class="pre">Zuckerkandl</span> <span class="pre">and</span> <span class="pre">Pauling</span> <span class="pre">(1965)</span></tt></a>  (pages 111-112) suggest:</p>
<blockquote>
<div><em>Deletions or additions of one to several amino acid residues are expected to be eliminated by natural selection in a high proportion of cases. Those that are preserved should be mostly found at either end of a chain, at the end of helices, in short helices, or in nonhelical regions, notably in loops that may be shortened or lengthened without  affecting the steric relationships in the rest of the molecule, A deletion or additionin the middle of a long helix would result in so many simultaneous alterations in side-chain interactions that it is highly unlikely that the tertiary structure and the function of the molecule could survive such an event. The deletions or additions found in hemoglobin and myoglobin chains are compatible with these generalities.</em></div></blockquote>

</article>


</section>

<section id="slide_notes">

</section>

  </body>
</html>